azure-ipam/main.go

```diff
@@ -4,6 +4,7 @@ import (
 	"log"
 	"os"
 
+	"github.com/Azure/azure-container-networking/azure-ipam/internal/buildinfo"
```

Comment:
should we just use the cni buildinfo pkg instead?

Reply:
I don't think azure cni has a buildinfo package currently, unless you are talking about a different cni?

Reply:
`github.com/containernetworking/plugins/pkg/utils/buildversion.BuildVersion`

Reply:
hmm when I replaced with that pkg, the version became an empty string